src/keri/core/eventing.py

```diff
@@ -5956,7 +5956,7 @@ def processEscrowPartialDels(self):
                 sigs = self.db.getSigs(dgkey)  # list of sigs
                 if not sigs:  # empty list
                     # no sigs so raise ValidationError which unescrows below
-                    msg = f"PDE Missing escrowed evt sigs at dig = {bytes(edig)}"
+                    msg = f"PDE Missing escrowed evt sigs at dig = {edig}"
                     logger.info("Kevery unescrow error: %s", bytes(edig))
```

Comment:
You either want `msg` or `edig` here but `bytes(edig)` is wrong.

Reply:
yes, the `bytes(edig)` is a holdover from the old logging statements. For example, from KERIpy 1.1.25 the logging statements Sam wrote look like this:
```python
logger.info("Kevery unescrow error: Missing event sigs at."
                                    "dig = %s\n", bytes(edig))
```

Reply:
f-strings are capable to convert the `edig` to the `__repr__` format for the given datatype, whatever it is.

Reply:
The reason line 5960 does not cause an error is because the log substitution with the `%s` operator auto-converts to the `__repr__` of whatever data type is there, thus a string encoding is not needed. All the old logging statements used `%s` and log substitutions, not f-string interpolations.

The problem I am trying to fix is when the call to `bytes(edig)` fails due to the lack of an encoding. It just so happens that this is occurring inside of an f-string.

Reply:
okay, I am wrong, you are right. The `bytes(edig)` call would cause an error if it were a string.

Reply:
My latest commit addresses the underlying issue by me going and doing the work I should have done the first time around, reading all of the Baser or LMDBer functions to see their specific return types and then ensuring the type used in the log statement is correct.